### DISCUSSION TITLE
Invoice Address block browser moving

### DISCUSSION THREAD

**Discussion by kidexx** [kimai/kimai#4268] [Questions & Answers]
Hi,
I have the following issue / the address block is moving in the browser printing:

[![2023-09-01-091750.png](https://i.postimg.cc/NFhjKfhQ/2023-09-01-091750.png)](https://postimg.cc/r0JkbTFH)

Any Ideas how to change it? I need to change the default CSS or? / My installation is just the default docker image.

And, but minor, when printing aggregated by project, is it possible to remove in the printing the date field in the lines (it comes from the first occurance of the invoice position.

**Comment by kevinpapst** [comment]
> Any Ideas how to change it? 

The template uses the invoice CSS https://github.com/kimai/kimai/blob/main/templates/invoice/renderer/invoice.html.twig#L7
which is located here: https://github.com/kimai/kimai/blob/main/assets/sass/_invoice.scss

But changing these rules require to rebuild them... so not easy without the fully installed stack.

Why not using the PDF version?

> And, but minor, when printing aggregated by project, is it possible to remove in the printing the date field in the lines (it comes from the first occurance of the invoice position.

Use a different template. PDF has two versions and "Leistungsdatum" is what you are looking for:
<img width="226" alt="Bildschirmfoto 2023-09-01 um 12 09 19" src="https://github.com/kimai/kimai/assets/533162/d678d05d-0b77-43ad-b559-37e32e8f5357">

Or copy & paste the original HTML version and adapt it to your needs.
